molo/core/styles/modules/_modules.scss

```diff
@@ -0,0 +1,52 @@
+
+@import "_m-typography";
+@import "_m-featured-latest-articles";
+@import "_m-buttons";
+@import "_m-back-to-top";
+@import "_m-banner";
+@import "_m-category-list";
+@import "_m-colors";
+@import "_m-pagination";
+@import "_m-tags";
+@import "_m-search";
+
+@import "header/_m-languages";
+@import "header/_m-languages-horizontal";
+@import "header/_m-languages-dropdown";
+@import "header/_m-header";
+@import "header/_m-logo";
+
+@import "menu/_m-menu-search-toggle";
+@import "menu/_m-menu-vertical-submenu";
+@import "menu/_m-menu";
+@import "menu/_m-navigation";
+
+@import "breadcrumbs/_m-breadcrumbs-on-footer";
+@import "breadcrumbs/_m-breadcrumbs";
+
+@import "forms/_m-forms";
+
+@import "articles/_m-article-hero";
+@import "articles/_m-article-images";
+@import "articles/_m-article-meta";
+@import "articles/_m-article";
+@import "articles/_m-featured-article";
+
+@import "comments/_m-comment-profile";
+@import "comments/_m-comment-success";
+@import "comments/_m-comments";
+
+@import "polls/_m-poll-image";
+@import "polls/_m-poll-right";
+@import "polls/_m-poll";
+
+@import "profiles/_m-profile-form";
+@import "profiles/_m-profile-view";
+@import "profiles/_m-profile";
+
+@import "yourwords/_m-yourwords-article";
+@import "yourwords/_m-yourwords-secondary";
+@import "yourwords/_m-yourwords";
```

Comment:
can we remove these plugins import so we won't forget to remove then and we also won't add anything related to the plugins in molo.core